Jenkinsfile

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-@Library('jenkinsfile-lib')_
+@Library('jenkinsfile-lib@PAL-824-k8s-example-smoketest')_
```

Comment:
Needs changing before merge, but after merge of BB changes. Is there a better way of doing this?

Reply:
Yeah, should be no problem to merge BB first